.ci/run-e2e-tests.sh

```diff
@@ -39,6 +39,16 @@ elif [ "${TEST_GROUP}" = "examples2" ]
 then
     echo "Running Examples2 Tests"
     make e2e-tests-examples2
+elif [ "${TEST_GROUP}" = "es-token-propagation" ]
+then
+    echo "Running token propagation tests"
+    oc create user user-test-token
+    oc adm policy add-cluster-role-to-user cluster-admin user-test-token
```

Comment:
The bad thing about using "root-like" permissions in a test is that you might miss the cases where the user might not have the appropriate permissions. Not sure it's relevant for your test here, but in general, we should aim to have the minimum set of permissions possible.